website/content/docs/commands/license/inspect.mdx

```diff
@@ -0,0 +1,82 @@
+---
+layout: docs
+page_title: 'Commands: license inspect'
+description: |
+  The license inspect command is used to inspect and validate an enterprise license.
+---
+
+# Command: license inspect
+
+<EnterpriseAlert>
+  This command is only present in Nomad Enterprise.
+</EnterpriseAlert>
+
+The `license inspect` command is used to inspect and validate a Nomad Enterprise
+license.  It will read the license from a file on disk or an environment variable
```

Comment:
```suggestion
license. It will read the license from a file on disk or an environment variable
```

Reply:
lol my muscle memory from like middle school still always types double spaces like this 😭